README.md

```diff
@@ -741,6 +741,14 @@ Get the url to connect to with
 
     minikube service jenkins --namespace kubernetes-plugin --url
 
+## Running in AWS EKS
+
+EKS enforces authentication to the cluster through [aws-iam-authenticator](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html). The token expires after 15 minutes
```

Comment:
This is only relevant if the Jenkins master is _not_ running in the cluster using a service account, which I would expect to be the most common mode. That should be made clearer.

Please also read, and refer to, [JENKINS-58143](https://issues.jenkins-ci.org/browse/JENKINS-58143) which discusses this flag. Ideally no such tricks would be necessary. The Kubernetes client we use _does_ support exec authentication such as is needed to use `aws-iam-authenticator` automatically from your kubeconfig, but last I checked it does not yet support automatic reauthentication after token expiry the way the Golang client does (for example).

Reply:
Ah, I added the docs in the wrong place then. In our use case, the Jenkins master is running an on EC2 instances out of k8s. 

I added this particular bit of documentation because, while it does initially authenticate using the `aws-iam-authenticator` correctly it does not re-authenticate later (as you stated it would not), however, setting the client cache expiry to something short seems to make it create a new client and thus reauthing.